swagger.json

```diff
@@ -3119,6 +3196,215 @@
                     }
                 }
             }
+        },
+        "Asset": {
+            "type": "object",
+            "description": "Basic information of an asset",
+            "required": [
+                "id"
+            ],
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "Asset ID",
+                    "format": "int64",
+                    "example": 1
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Asset name",
+                    "example": "Trailer 123"
+                },
+                "cable": {
+                    "type": "array",
+                    "description": "The cable connected to the asset",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "assetType": {
+                                "type": "string",
+                                "description": "Asset type",
+                                "example": "Reefer (Thermo King)"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "AssetReeferResponse": {
+            "type": "object",
+            "description": "Reefer-specific asset details",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "Asset ID",
+                    "example": 1
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Asset name",
+                    "example": "Reefer 123"
+                },
+                "assetType": {
+                    "type": "string",
+                    "description": "Asset type",
+                    "example": "Reefer (Thermo King)"
+                },
+                "reeferStats": {
+                    "type": "object",
+                    "properties": {
+                        "alarms": {
+                            "type": "array",
+                            "description": "Reefer alarms",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "changedAtMs": {
+                                        "type": "integer",
+                                        "format": "int64",
+                                        "description": "Timestamp in Unix milliseconds since epoch.",
```

Comment:
"The time in Unix epoch milliseconds when the alarms were last polled." hmm not sure what would be a more thorough explanation

Reply:
i'll change it to `Timestamp in Unix milliseconds since epoch when the alarms were reported.`